topics/imaging/tutorials/imaging-introduction/tutorial.md

```diff
@@ -0,0 +1,163 @@
+  - "How do I convert images using Galaxy?"
+  - "How do I display images in Galaxy?"
+  - "How do I filter images in Galaxy?"
+  - "How do I segment simple images in Galaxy?"
+objectives:
+  - "How to handle images in Galaxy."
+  - "How to perform basic image processing in Galaxy."
+requirements:
+time_estimation: "30M"
+
+contributors:
+  - thomaswollmann
+
+---
+
+# Introduction
+{:.no_toc}
+
+This tutorial shows how to use Galaxy for image analysis. The data used in this tutorial is available at [Zenodo](https://zenodo.org/record/3358874#.XUQ9oi1XZUM).
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Getting data
+
+The dataset required for this tutorial contains a screen of DAPI stained HeLa nuclei ([more information](https://zenodo.org/record/3358874#.XUQ9oi1XZUM)). We will use a sample image from this dataset for training basic imaging skills in Galaxy.
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. If you are logged in, create a new history for this tutorial
+>
+>    {% include snippets/create_new_history.md %}
+>
+> 2. Import the following dataset and choose the type of data as `zip`.
+>
+>    ```
+>    https://zenodo.org/record/3358874/files/20190919_MIA_Image_Data_siRNA_Screen.zip
+>    ```
+>
+>    {% include snippets/import_via_link.md %}
+>
+> 3. "Use the unzip tool to extract a single image from the zipped screen": `unzip`
+>    - {% icon param-file %} *"input_file"*: `Zipped ` input file
+>    - *"Extract single file"*: `Single file`
+>    - *"Filepath"*: `B2--W00026--P00001--Z00000--T00000--dapi.tif`
+>
+> 4. Rename dataset to `input.tif`
+>
+>    {% include snippets/rename_dataset.md %}
+
+{: .hands_on}
+
+
+# Image Metadata Extraction
+
+Now, we can extract metadata from an image.
+
+> ### {% icon hands_on %} Hands-on: Extract Image Metadata
+>
+> 1. **Image Info** {% icon tool %} with the following parameters to extract metadata from the image:
+>    - {% icon param-file %} *"Input Image"*: `input.tif` file (output of the previous step)
+> 2. Click on the {% icon galaxy-eye %} (eye) icon next to the file name, to look at the file content and search for image acquisition information
+>    ![eye](../../images/eye-icon.png){:width="320px"}
+{: .hands_on}
+
+# Image Convertion
+
+Not all tools can handle all image formats. Especially proprietary microscope image formats should be converted to TIFF ([supported formats](https://docs.openmicroscopy.org/bio-formats/5.7.1/supported-formats.html)). However, TIFF can not be displayed in the browser. Therefore, we convert `input.tif` to a PNG for visualization.
+
+> ### {% icon hands_on %} Hands-on: Convert Image
+>
+> 1. **Convert image** {% icon tool %} with the following parameters to convert the image to PNG:
+>    - {% icon param-file %} *"Input Image"*: `input.tif` file (output of the previous step)
+>    - *"Output data type"*: `PNG`
+> 2. Rename the generated file to `viz_input`
```

Comment:
you could add `{% include snippets/rename_dataset.md %}` on a line below this to remind people how to rename a dataset?

Reply:
oh, I see now that you already did this in an earlier box, that is probably good enough :)

Reply:
@shiltemann I tried to address your comments in the last commit.